compiler/circle-opselector/CMakeLists.txt

```diff
@@ -12,4 +12,6 @@ target_link_libraries(opselector luci_export)
 target_link_libraries(opselector arser)
 target_link_libraries(opselector vconone)
 
-install(TARGETS opselector DESTINATION bin)
+target_link_libraries(opselector luci_service)
+
+install(TARGETS opselector DESTINATION bin)
```

Comment:
@Paran-Yu Please fix "No newline at end of file". If you're using `VSCode`, you may let VSCode do for you. Refer to [SO](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files).